deployment/deployment.yaml.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage_if_empty() {
+  if [[ -z "${1}" ]]
+  then
+      usage
```

Comment:
Definition of a func below it's use will not work I think

Reply:
it hasn't been an issue. but moved it after